api-server/api/openapi.yaml

```diff
@@ -178,4 +212,27 @@ components:
       - eventId
       title: A Ceramic Event
       type: object
+    EventFeed:
+      description: Ceramic event keys as part of a Ceramic Stream
+      example:
+        resumeToken: resumeToken
+        events:
+        - eventId: eventId
```

Comment:
nitpick: should we just call the fields in individual Events `id` and `data`, instead of `eventId` and `eventData`?  Save some bytes in the responses?

Reply:
hmm. yeah, probably. I didn't do that thinking it'd breaks all the existing routes, however it looks like we use this as the POST event payload and the response for the subscribe endpoint, which is now deprecated. I made a new type called `Event` that is just `id, data` , and renamed the existing one `EventDeprecated`.

Reply:
implemented in [f33455b](https://github.com/ceramicnetwork/rust-ceramic/pull/255/commits/f33455b25cedaf8ab191eb706e195817a441b88c)

Reply:
also updated `events/$eventId` to use this new format after merging main (#258)